Ascend.java

```diff
@@ -2,6 +2,22 @@
 // and prints them in increasing order.
 public class Ascend {

```

Comment:
OVERALL:
It is good submission. Logic was mainly good. 2 cases which I want to point out:
1. TimeFormat.java -> There was duplication of if statements. this code is harder to maintain, please see improvement suggestions
2. There was redundant split of argument and the parsing in FVCalc.java
3. There was redundant casting of int to int. in NumWords.java

Styling wise, it was ok, spacing and indentations were the main issues, please go over styling guidelines.

Reply:
Thank you!
I will use that for my future codes.